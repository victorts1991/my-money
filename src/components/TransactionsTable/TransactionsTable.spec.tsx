import { render } from "@testing-library/react"
import { TransactionsTable } from "."

test("TransactionsTable renders correctly", () => {
    render(<TransactionsTable />)
})