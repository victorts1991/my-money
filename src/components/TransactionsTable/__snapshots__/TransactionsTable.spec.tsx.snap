// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TransactionsTable mock useEffect when loading transactions 1`] = `ShallowWrapper {}`;
